docs/fundamentals/executor/executor-api.md

```diff
@@ -147,7 +147,7 @@ class MyExecutor(Executor):
     @requests
     async def foo(
         self, docs: DocumentArray, parameters: Dict, docs_matrix: List[DocumentArray]
-    ) -> Optional[Union[DocumentArray, Dict]]:
+    ) -> Union[DocumentArray, Dict, None]:
```

Comment:
```suggestion
    ) -> Optional[Union[DocumentArray, Dict]]:
```

This is by definition Optional and is much more standard

Reply:
actually it is the oppsite, if you look `typing`, `pydantic` or dataclass, they all get translated into `Union[..., None]`

Reply:
Here it is better map to what I later said, "there are 3 ways of returning", `Union[1,2,3] -> 3 ways`